content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: documentation
+title: Upgrading Jenkins Java version from 8 to 11
+---
+
+In order to upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11, there are few details and steps to take care.
+
+First, we need to understand that some plugins were not directly compatible with Java 11. 
+This means that the community had to fix those plugins and release them.
+So, we need to make sure that, if those plugins are installed in our instance, we need to make sure they are installed with those versions or more recent ones.
+
+
+== Workflow Support 3.0 
+
+One of the most important upgrade is about the plugin:workflow-support[Workflow Support plugin].
```

Comment:
One of the most important upgrades is the plugin:workflow-support[Workflow Support plugin].

Reply:
```suggestion
One of the most important upgrades is the plugin:workflow-support[Workflow Support plugin].
```